README.md

```diff
@@ -15,16 +15,17 @@
 
 ## Live Demo
 
-[Live Demo Link](https://keroloussamy.github.io/Leaderboard/dist/)
+[Live Demo Link](https://keroloussamy.github.io/math-magicians/)
 
 
 ## Getting Started
 To get a local copy up and running follow these simple example steps.
 
 - Click on the top right green "code" button.
 - On the dropdown menu, choose "download with zip" button.
-- After download, extract the zip file and you have the project running on your machine.
-
+- After download, extract the zip file and you have the project on your machine.
+- After downloading run `npm i` to install all Node packages.
+- Run `npm start` to run the project.
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Done